extensions/exposition/schemas/octets/store.cos.yaml

```diff
@@ -1,4 +1,25 @@
-accept+: string
-workflow+: <string>
-trust: [string]
-_: true
+type: object
+nullable: true
+properties:
+  accept:
+    anyOf:
+      - type: string
+      - type: array
+        items:
+          type: string
+  limit:
+    type: string
+    pattern: ^(\d+(\.\d+)?)([kmgtKMBGT]i?)?[bB]?$
```

Comment:
Shouldn't it be `anyOf` or `oneOf` and allow `integer` ?

Reply:
RegExp matches numbers. Making this `anyOf` will make internal application types unnecessary redundant.